content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md

```diff
@@ -0,0 +1,33 @@
+---
+title: monitor.stateChangesOnly() function
+description: >
+  The `monitor.stateChangesOnly()` function  takes a stream of tables that contains a` _level`
```

Comment:
"stream of tables" same as "stream of data"?

Reply:
curious what is distinction between `monitor.stateChanges` and `monitor.stateChangesOnly`

Reply:
1. Yes, they're the same. I updated them so they match.
2. `monitor.stateChanges` gives you to option of specifying which changes to detect (`any` to `crit`, `warn` to `info`, etc.). `monitor.stateChangesOnly` simply outputs rows where the state changes. It's actually used in the function definition for `monitor.stateChanges`.